src/context/context.ts

```diff
@@ -0,0 +1,33 @@
+import { ChainMap, ChainMetadata, MultiProtocolProvider } from '@hyperlane-xyz/sdk';
+
+import type { RoutesMap } from '../features/routes/types';
+import type { TokenMetadata } from '../features/tokens/types';
+
+import Chains from './_chains.json';
+import Routes from './_routes.json';
+import Tokens from './_tokens.json';
+
+export interface WarpContext {
+  chains: ChainMap<ChainMetadata & { mailbox?: Address }>;
+  tokens: TokenMetadata[];
+  routes: RoutesMap;
+  multiProvider: MultiProtocolProvider<{ mailbox?: Address }>;
+}
+
+let warpContext: WarpContext;
+
+export function getWarpContext() {
+  if (!warpContext) {
+    warpContext = {
+      chains: Chains as any,
```

Comment:
`any` cast is required because typescript isn't smart enough to recognize enums in the serialized data as the valid enum values. It's a safe cast because all this data gets run through zod at build time

Reply:
even with `resolveJsonModule` on the tsconfig?

Reply:
Yep. There maybe we a way to force it to work but tbh didn't see worth the trouble